test/functional/interface_ipc.py

```diff
@@ -256,9 +256,11 @@ async def interrupt_wait():
             assert_equal(res.result, True)
 
             self.log.debug("Block should propagate")
-            assert_equal(self.nodes[1].getchaintips()[0]["height"], current_block_height + 1)
+            # Check that the IPC node actually updates it own chain
+            assert_equal(self.nodes[0].getchaintips()[0]["height"], current_block_height + 1)
```

Comment:
it own -> its own [possessive pronoun "its" required; "it" is incorrect here]

drahtbot_id_5_m